content/docs/setup/kubernetes/_index.md

```diff
@@ -2,13 +2,59 @@
 title: Kubernetes
 description: Instructions for installing the Istio control plane on Kubernetes and adding virtual machines into the mesh.
 weight: 10
-keywords: [kubernetes]
+type: section-index
 aliases:
     - /docs/tasks/installing-istio.html
     - /docs/setup/install-kubernetes.html
 icon: kubernetes
+keywords: [kubernetes, install, quick start, setup, installation]
 ---
 
 {{< tip >}}
 Istio {{< istio_version >}} has been tested with these Kubernetes releases: {{< supported_kubernetes_versions >}}.
 {{< /tip >}}
+
```

Comment:
How about moving the following contents from here into a new (first) sub document (i.e., `/docs/setup/kubernetes/quick-start` or `/docs/setup/kubernetes/getting-started`)?

Reply:
I prefer not to do that because that means that readers need to know that the landing page is there and explicitly click it. This way, whenever someone uses the left navigation and clicks on Kubenetes as part of that navigation, they get the guidance needed to know what path to follow. Which is the intent of the PR.

Reply:
How about putting a section heading here:
```
## Getting started
```
Seems that there should be a clear indication of where to start. You could also link to this section directly from the landing page: `/doc/setup/kubernetes/#getting_started`

Reply:
I'm fine with adding the heading, sans the gerund. :smiley: